pkg/archive/changes.go

```diff
@@ -386,7 +386,21 @@ func ExportChanges(dir string, changes []Change) (Archive, error) {
 		// during e.g. a diff operation the container can continue
 		// mutating the filesystem and we can see transient errors
 		// from this
+	changesLoop:
```

Comment:
Could be my old-school mindset (and I know goto's are ok in most languages) but it gives me the heebie-jeebies  :-)  Just saying...